packages/react-aria-components/test/AriaAutocomplete.test-util.tsx

```diff
@@ -970,7 +970,7 @@ export const AriaAutocompleteTests = ({renderers, setup, prefix, ariaPattern = '
 
           // Open the nested submenu
           await user.keyboard('{ArrowDown}');
-          await user.keyboard('{ArrowDown}');
+          // await user.keyboard('{ArrowDown}');
```

Comment:
was an item removed?

Reply:
this is because we autofocus the menu now, so there's one less arrow down to do